### PR TITLE
KFSPTS-11759 KFSPTS-10229 customize cgb/ar pages to use CuAward instead of Award

### PR DESCRIPTION
**Pull request by db483**
This is a fix we have had in the CGB environment and is needed in develop for turning on CGB in DEV.

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/module/ar/contractsGrantsInvoiceSummaryResultContent.tag [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<%--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2017 Kuali, Inc.
```
Comment (cah292) [inline src/main/webapp/WEB-INF/tags/module/ar/contractsGrantsInvoiceSummaryResultContent.tag]: License header is out-of-date.

File: src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<%--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2017 Kuali, Inc.
```
Comment (cah292) [inline src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp]: License header is out-of-date.

File: src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+            <tr>
+                <th>
+                    <label>
+                        <kul:htmlAttributeLabel attributeEntry="${award.proposalNumber}" readOnly="true"/>
+                    </label>
+                </th>
+                <td>
+                    <kul:htmlControlAttribute attributeEntry="${award.proposalNumber}" property="proposalNumber"/>
+                    <kul:lookup boClassName="org.kuali.kfs.module.cg.businessobject.Award"/>
```
Comment (cah292) [inline src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp]: Should this lookup reference CuAward as well?

Comment (db483) [reply src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp]: I've checked and the lookup works fine. We also did not have a report of a problem with this. I am not sure it needs to be updated.

**Comment by db483** [comment]
Looks like the license headers have changed since this change was put in place. I have updated them. Thank you!

File: src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+          headerTitle="Federal Financial Report Generation"
+          docTitle="Federal Financial Report Generation" renderMultipart="true"
+          transactionalDocument="false" htmlFormAction="arFederalFinancialReport"
+          errorKey="foo">
+
+    <div id="lookup">
+        <div class="main-panel">
+            <div class="headerarea-small"></div>
+            <table class="standard" style="margin: 20px auto 0 auto;" summary="Federal Financial Report Generation"
+            ">
```
Comment (cah292) [inline src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp]: This quotation mark doesn't have a corresponding match. The base file from KualiCo already has this problem, so I'm fine with moving this forward as-is and then we can create a ticket to have KualiCo fix it.

**Pull request merged**